### PR TITLE
Allow bootstrap to resume when pivotal key already exists

### PR DESCRIPTION
**Pull request by marcparadise**
I need to do some testing on this, but I believe this will resolve https://github.com/chef/chef-server/issues/987.

### CODE REVIEW

**Comment by marcparadise** [comment]
I'll need to fix this history too.

**Comment by stevendanna** [comment]
Overall this looks good to m.

File: omnibus/files/private-chef-cookbooks/private-chef/recipes/private_keys.rb [hunk 1]
```diff
@@ -24,34 +24,40 @@
...
   sensitive true
 end
 
 webui_priv_key_path = "/etc/opscode/webui_priv.pem"
-new_webui = !File.exists?(webui_priv_key_path)
-webui_key = OpenSSL::PKey::RSA.generate(2048) if new_webui
+webui_key = if File.exists?(webui_priv_key_path)
+                OpenSSL::PKey::RSA.new(File.read(webui_priv_key_path))
+            else
+                webui_key = OpenSSL::PKey::RSA.generate(2048) if new_webui
```
Comment (stevendanna) [inline omnibus/files/private-chef-cookbooks/private-chef/recipes/private_keys.rb]: no `webui_key = ` and no `if new_webui` here

File: omnibus/files/private-chef-cookbooks/private-chef/recipes/private_keys.rb [hunk 2]
```diff
@@ -24,34 +24,40 @@
...
+            end
 
-file "/etc/opscode/webui_pub.pem" do
+  file "/etc/opscode/webui_pub.pem" do
```
Comment (stevendanna) [inline omnibus/files/private-chef-cookbooks/private-chef/recipes/private_keys.rb]: odd spacing here.

**Comment by stevendanna** [comment]
@marcparadise Thoughts on moving this forward?  This is still my best hunch for what users are hitting in the field.

**Comment by marcparadise** [comment]
Testing notes: if you pre-create /etc/opscode/pivotal.pem then run the first chef-server-ctl reconfigure, this underlying scenario should occur.

**Pull request closed without merging**